framework/web/handler.go

```diff
@@ -45,6 +52,34 @@ func init() {
 	}
 }
 
+func (e *panicError) Error() string {
+	return e.err.Error()
+}
+
+func (e *panicError) String() string {
+	return e.err.Error()
+}
+
+func (e *panicError) Unwrap() error {
+	return e.err
+}
+
+func (e *panicError) Format(s fmt.State, verb rune) {
+	switch verb {
```

Comment:
Is it usual to have no default formatting?
And should we document the verbs and flags at least in the function's doc block?

Reply:
This is the `fmt.Formatter` interface. It is based on `github.com/pkg/errors`, and not used anywhere outside of the flamingo frameowrk. I would keep it like this until we find a better way for backtraces.